app/services/oneoffs/flipflop_users_cleanup.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Oneoffs
+  class FlipflopUsersCleanup
+    def call
+      result = []
+
+      flip_flop_users.each do |from_user|
+        if from_user.archived?
+          result << [from_user.id, "User is archived"]
+          next
+        end
+
+        from_trn = from_user.teacher_profile.trn
+        if from_trn.blank?
+          result << [from_user.id, "TRN does not exist"]
+          next
+        end
+
+        to_user = primary_user(from_trn)
+
+        if from_user == to_user
+          result << [from_user.id, "already the primary user"]
+          next
+        end
+
+        if to_user.archived?
+          result << [from_user.id, "to_user [#{to_user.id}] is archived"]
+          next
+        end
+
+        Identity::Transfer.call(from_user:, to_user:)
+        result << [from_user.id, "transfer to user [#{to_user.id}]"]
+      end
+
+      result
+    end
+
+    def primary_user(trn)
```

Comment:
This is in prod now https://github.com/DFE-Digital/early-careers-framework/blob/main/app/services/identity/primary_user.rb#L6